src/java.se/share/data/jdwp/jdwp.spec

```diff
@@ -98,7 +98,7 @@ JDWP "Java(tm) Debug Wire Protocol"
         "The returned list contains a threadID for each live "
         "<a href=../../api/java.base/java/lang/Thread.html#platform-threads>platform thread</a> "
         "in the target VM. This includes platform threads created with the Thread "
-        "API and and all native threads attached to the target VM with JNI code."
+        "API and all native threads attached to the target VM with JNI code."
```

Comment:
The spec for the JDWP AllThreads command was significantly reworded in Java 19 so this is where this typo crept in. We have JDK-8294672 tracking it to fix for Java 20, maybe you should take it?

Reply:
Since this PR has gone stale, I'll be fixing this typo in jdwp.spec via [JDK-8294672](https://bugs.openjdk.org/browse/JDK-8294672).